packages/mui-nested-menu/src/components/nestedMenuItemsFromObject.tsx

```diff
@@ -33,6 +33,7 @@ export function nestedMenuItemsFromObject({
                     parentMenuOpen={isOpen}
                     sx={sx}
                     disabled={disabled}
```

Comment:
Seeing as we are already passing down `disabled` which contains the same value as `parentMenuDisabled`, perhaps we use `MenuItemProps.disabled` within `NestedMenuItem` to apply the new logic?

Reply:
Ill remove this change seeing as there is no activity.